R/eventemitters.R

```diff
@@ -82,9 +82,9 @@ EventEmitter <- R6::R6Class(
         tryCatch(callbacks$invoke(...),
                 error = function(e) self$emit("error", e)
         )
-        invisible(self)
+        TRUE
       } else {
-        invisible(self)
+        FALSE
```

Comment:
Return `self` was to allow piping
```r
emit(...)$emit(...)
```
Some package does that with R6 class and I find it useful.
It is maybe more important to stick to eventemitter nodejs definition. 🤔

Reply:
I like this way too.
I will revert this modification

Reply:
done